apps/archive/commands.py

```diff
@@ -48,6 +49,17 @@ def run(self):
 
         remove_locks()
 
+    def _remove_expired_publish_queue_items(self):
+        system_settings_service = get_resource_service('system_settings')
+        expire_interval = system_settings_service.find_one(req=None, name='publish_queue_expire_interval')
+        if not expire_interval:
+            return
+
+        expire_time = utcnow() - system_settings_service.validate_timedelta(expire_interval)
+        logger.info('{} Removing publish queue items created before {}'.format(self.log_msg, str(expire_time)))
+
+        get_resource_service('publish_queue').delete({'_created': {'$lte': expire_time}})
```

Comment:
Can you define the index on the `_created` field for the `publish_queue` resource?

Reply:
or you can use something like `{'_id': {'$lte': ObjectId.from_datetime(expire_time)}}`

Reply:
I thought that index might not exists but we already have index on `_created`.